### PR TITLE
Trivy local setup document update

### PR DESCRIPTION
**Pull request by ru-ukho**

### CODE REVIEW

File: trivy/README.md [hunk 1]
```diff
@@ -46,6 +46,21 @@ On rare occasions, you might find it necessary to omit specific checks, although
 
 Trivy will generate and publish results in the build pipeline under the Test and Coverage sections, allowing you to access a comprehensive overview of successes and failures. Utilise the filter to seamlessly switch between the two views. Furthermore, any failed scans will be clearly presented within the Docker console for immediate visibility and action.
 
+## local setup
```
Comment (bharathianjeneya) [inline trivy/README.md:49]:
```suggestion
## Local setup
```

Comment (elerivaliant) [inline trivy/README.md:49]:
```suggestion
## Local setup
```

File: trivy/README.md [hunk 2]
```diff
@@ -46,6 +46,21 @@ On rare occasions, you might find it necessary to omit specific checks, although
...
+
+Trivy can be installed locally by downloading the windows compatible binaries [from](https://github.com/aquasecurity/trivy/releases/latest/) and updating binary files location at `PATH` system environment variable.
```
Comment (elerivaliant) [inline trivy/README.md:51]:
```suggestion
Trivy can be installed locally on Windows by downloading the [Windows compatible binaries](https://github.com/aquasecurity/trivy/releases/latest/) and adding the location of the files to the Windows PATH environment variable.
```

File: trivy/README.md [hunk 3]
```diff
@@ -46,6 +46,21 @@ On rare occasions, you might find it necessary to omit specific checks, although
...
+Running trivy scan at command prompt locally as below:
```
Comment (elerivaliant) [inline trivy/README.md:52]:
```suggestion

Once installed, run a Trivy scan using a command prompt be navigating to your Trivy location and running commands as below:
```

File: trivy/README.md [hunk 4]
```diff
@@ -46,6 +46,21 @@ On rare occasions, you might find it necessary to omit specific checks, although
...
+
+- To scan a local project including language-specific files:
+`trivy fs /path/to/your_project_folder`
+
+- To scan a single file:
+`trivy fs ./trivy-ci-test/single_file`,
+
+Other operating systems installation instructions can be found [here](https://aquasecurity.github.io/trivy/v0.18.3/installation/)
```
Comment (elerivaliant) [inline trivy/README.md:60]:
```suggestion
Other operating system installation instructions can be found [here].(https://aquasecurity.github.io/trivy/v0.18.3/installation/)
```

File: trivy/README.md [hunk 5]
```diff
@@ -46,6 +46,21 @@ On rare occasions, you might find it necessary to omit specific checks, although
...
+
+Instructions to use Trivy in [VS Code plugin](https://github.com/aquasecurity/trivy-vscode-extension)
```
Comment (elerivaliant) [inline trivy/README.md:62]:
```suggestion
A [VS Code plugin](https://github.com/aquasecurity/trivy-vscode-extension) for Trivy is available.
```

**Review by bharathianjeneya — APPROVED** [review]

File: trivy/README.md [hunk 6]
```diff
@@ -46,6 +46,22 @@ On rare occasions, you might find it necessary to omit specific checks, although
 
 Trivy will generate and publish results in the build pipeline under the Test and Coverage sections, allowing you to access a comprehensive overview of successes and failures. Utilise the filter to seamlessly switch between the two views. Furthermore, any failed scans will be clearly presented within the Docker console for immediate visibility and action.
 
+## Local setup
+
+Trivy can be installed locally on Windows by downloading the [Windows compatible binaries](https://github.com/aquasecurity/trivy/releases/latest/) and adding the location of the files to the Windows PATH environment variable.
+
+Once installed, run a Trivy scan using a command prompt be navigating to your Trivy location and running commands as below:
```
Comment (elerivaliant) [inline trivy/README.md:53]:
```suggestion
Once installed, run a Trivy scan using a command prompt by navigating to your Trivy location and running commands as follows:
```

File: trivy/README.md [hunk 7]
```diff
@@ -46,6 +46,22 @@ On rare occasions, you might find it necessary to omit specific checks, although
...
+
+- To scan a local project including language-specific files:
+`trivy fs /path/to/your_project_folder`
+
+- To scan a single file:
+`trivy fs ./trivy-ci-test/single_file`,
+
+Other operating system installation instructions can be found [here].(https://aquasecurity.github.io/trivy/v0.18.3/installation/)
```
Comment (elerivaliant) [inline trivy/README.md:61]:
```suggestion
Other operating system installation instructions can be found [here](https://aquasecurity.github.io/trivy/v0.18.3/installation/).
```

**Review by elerivaliant — APPROVED** [review]

**Review by ShaunBresnahan — APPROVED** [review]
Happy with advise

**Pull request merged**